integration/update_field_compat_test.go

```diff
@@ -55,14 +55,14 @@ func TestUpdateFieldCompatUnset(t *testing.T) {
 	testCases := map[string]updateCompatTestCase{
 		"Simple": {
 			update:        bson.D{{"$unset", bson.D{{"v", ""}}}},
-			skipForTigris: true,
+			skipForTigris: "TODO",
```

Comment:
Should we create an issue for these `TODO`s?

Reply:
Yes, but probably not right now